File: hotel_api/hotel_data/dto/UserDto.cs
using System;

namespace hotel_data.dto;

public class UserDto
{
    public UserDto(Guid userId, Guid personID, DateTime brithDay, bool isVip, PersonDto personData, string userName, string password)
    {
        this.userId = userId;
        this.personID = personID;
        this.brithDay = brithDay;
        this.isVip = isVip;
        this.personData = personData;
        this.userName = userName;
        this.password = password;
    }

    public Guid userId  { get; set; }
    public Guid personID  { get; set; }
    public DateTime brithDay   { get; set; }
    public bool isVip   { get; set; }
    public PersonDto personData { get; set; }
    public string userName { get; set; }
    public string password { get; set; }
    
}

File: hotel_mobile/app/src/main/java/com/example/hotel_mobile/Dto/UserDto.kt
package com.example.hotel_mobile.Dto

import com.example.hotel_mobile.services.kSerializeChanger.LocalDateTimeKserialize
import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.datetime.LocalDateTime
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class UserDto(
    val userId: String? = null,
    val addBy: String? = null,  // Nullable UUID
    @Serializable(with = LocalDateTimeKserialize::class)
    val brithDay: LocalDateTime? = null,
    val isVip: Boolean? = false,
    val personData: PersonDto,
    val userName: String,
    val password: String,
    val isDeleted: Boolean,
    val imagePath: String? = null,
    val isUser: Boolean? = true
)